clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java

```diff
@@ -203,9 +218,13 @@ public String getApplicationName() {
     private final String environment;
     private final String accountType;
     private final String projectName;
+    private final boolean alphaListed;
     private final String jsonKey;
     private final List<String> imageProjects;
     private final GoogleCredentials credentials;
     private final List<String> requiredGroupMembership;
     private final String applicationName;
+
+    private static final String COMPUTE_SERVICE_PATH = "compute/v1/projects/";
+    private static final String COMPUTE_ALPHA_SERVICE_PATH = "compute/alpha/projects/";
```

Comment:
How do feel about making `v1`, `alpha`, etc an enum to switch between library versions? `v2` might be on the way, and perhaps there is a need for `beta`.